Makefile

```diff
@@ -1,12 +1,12 @@
 .PHONY: all
 all: test
 
-.PHONY: dep
-dep:
-	dep ensure -v
+.PHONY: mod
+mod:
+	go mod tidy && go mod vendor
 
 .PHONY: format
-format: dep
+format: mod
 	go fmt ./...
```

Comment:
add `-mod=vendor` flag

Reply:
add to vet and test as well...

Reply:
Thanks @tchughesiv 
Updated

Reply:
If it is added to the Makefile why add it to the travis file? We will be setting it twice which is not allowed.
I have added it only to the Makefile

Reply:
Sounds good for travis file... likely need to add to build command though.